src/extension.ts

```diff
@@ -55,6 +56,7 @@ export function activate(context: vscode.ExtensionContext) {
         console.log('one configuration settings...');
       });
   context.subscriptions.push(disposableOneConfigurationSettings);
+  createStatusBarItem(context);
```

Comment:
Could you please add a emty line between 
```
  context.subscriptions.push(disposableOneConfigurationSettings);

  createStatusBarItem(context);
```

Reply:
I fixed it. thank you.